### DISCUSSION TITLE
I am seeing this error, while try to run a mojo file , can any one help me to solve this problem?

### DISCUSSION THREAD

**Discussion by tedzz12** [modular/modular#1251] [General]
![Screenshot from 2023-11-07 20-54-36](https://github.com/modularml/mojo/assets/66946445/de910798-b860-4ca9-a8c3-f3c8b82b40e3)
i already installed mojo in Ubuntu 20.04 system,but i cannot find thew path, and i am seeing this error, when click open settings i also saw another  window for path.
![Screenshot from 2023-11-07 19-42-49](https://github.com/modularml/mojo/assets/66946445/6c2977ad-9845-4b88-a04c-10d074c17de7)

**Comment by Sharktheone** [comment]
You  need to set the path to the `.modular` directory for the plugin to work. For me it was `/home/shark/.modular` You need to replace `shark` with your username.

**Reply by tedzz12** [reply]
where do i find my folder where mojo is installed?

**Reply by Sharktheone** [reply]
Mojo lives in the `~/.modular` directory, under `pkg/packages.modular.com_mojo/` but to fix your error just paste the output of `echo $HOME/.modular` into the field for the Modular home path

**Reply by tedzz12** [reply]
I am using Ubuntu 20.04

**Reply by Sharktheone** [reply]
Yes, again, just paste the output of this command `echo $HOME/.modular` into the field for the Modular home path
